docs/aca/04-aca-dapr-stateapi/index.md

```diff
@@ -291,17 +310,39 @@ scopes:
 Few things to note about this yaml file:
 - We've used the name `statestore` which should match the name of statestore we've used in the `TaskStoreManager.cs` file. As well we have set the metadata key/value to allow us to connect to Azure Cosmos DB. You need to replace the `masterKey` value with your Cosmos Account key. Remember this is only needed for local development debugging, we will not be using the masterKey when we deploy to ACA. 
 - We've updated the other metadata keys such as `database`, `collection`, etc... to match the values of your Cosmos DB instance. For full metadata specs, you can check this [page](https://docs.dapr.io/reference/components-reference/supported-state-stores/setup-azure-cosmosdb/).
-- By default, all dapr-enabled container apps within the same environment will load the full set of deployed components. By adding `scopes` to a component, you tell the Dapr sidecars for each respective container app which components to load at runtime. Using scopes is recommended for production workloads. In our case, we have set the scopes to read `tasksmanager-backend-api` as this will be the application that needs access to Azure Cosmos DB State Store. More about scopes can be found on this [link](https://learn.microsoft.com/en-us/azure/container-apps/dapr-overview?tabs=bicep1%2Cyaml#component-scopes).
+- Replace the url value with the URI value of your cosmos database account. You can get that from the Azure portal by navigating to the cosmos database account overview page and get the uri value from there. Basically the uri should have the following structure. https://COSMOS_DB_ACCOUNT.documents.azure.com:443/.
+- By default, all dapr-enabled container apps within the same environment will load the full set of deployed components. By adding `scopes` to a component, you tell the Dapr sidecars for each respective container app which components to load at runtime. Using scopes is recommended for production workloads. In our case, we have set the scopes to `tasksmanager-backend-api` as this will be the application that needs access to Azure Cosmos DB State Store. More about scopes can be found on this [link](https://learn.microsoft.com/en-us/azure/container-apps/dapr-overview?tabs=bicep1%2Cyaml#component-scopes).
```

Comment:
Maybe rephrase to highlight that value `tasksmanager-backend-api` is the dapr app id not the container app name, we can use the following: 

we have set the scopes to `tasksmanager-backend-api` which represents the `dapr-app-id` which is associated to the container app that needs access to Azure Cosmos DB State Store.